src/data/text/item_descriptions.h

```diff
@@ -2005,9 +2030,9 @@ static const u8 sLifeOrbDesc[] = _(
     "of some HP per turn.");
 
 static const u8 sPowerHerbDesc[] = _(
-    "Allows the immediate\n"
-    "use of a move that\n"
-    "charges first.");
+    "It allows to immedi-\n"
```

Comment:
i prefer the original wording, i think "allows to immediately use" is a bit awkward